content/en/platform/corda/5.0/introduction/release-notes/5.0.0.md

```diff
@@ -10,3 +10,16 @@ menu:
 section_menu: corda5
 ---
 # Corda 5.0.0 Release Notes
+This is the release of the next generation of Corda. This major new release introduces the concept of a virtual node and it runs on a new architecture with a new technology stack to provide the foundations for running Corda at scale. 
+
+Corda 5.0 will be supported on a short-term support cycle and will be supported by R3 until a long-term support release is available. Refer to [End of Life Strategy]({{< relref "../about-the-docs.md#end-of-life-strategy">}}).
```

Comment:
```suggestion
Corda 5.0 will be supported on a short-term support cycle by R3 until a long-term support release is available. Refer to [End of Life Strategy]({{< relref "../about-the-docs.md#end-of-life-strategy">}}).
```

Reply:
Resolved by alternative suggestion for this line.